<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowBrakeMarkerEditor</class>
 <widget class="QDialog" name="WindowBrakeMarkerEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brake marker editor</string>
  </property>
  <widget class="QComboBox" name="cb_SetList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_SelBrMaSe">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selected brake marker set:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_CurrentSet">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>501</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>Current set:</string>
   </property>
   <widget class="QLabel" name="lbl_Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_CurrentSetName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>401</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Save">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Create new</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Rename">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Rename</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Delete">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lb_Markers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>311</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb_Marker">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>141</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Marker</string>
    </property>
    <widget class="QLabel" name="lbl_Distance">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Distance:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_Speed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed (kph):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_Distance">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_Speed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_AddNewMarker">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>136</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Add new</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_DelMarker">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete selected</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lbl_ExistingMarkers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Existing markers:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Track">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Track:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Track">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>321</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_SelectTrack">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Select track</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
